### PR TITLE
[6085] Item path is not displayed in dashlet

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/6085

### CODE REVIEW

File: ui/app/src/components/UnpublishedDashlet/UnpublishedDashlet.tsx [hunk 1]
```diff
@@ -279,6 +294,7 @@ export function UnpublishedDashlet(props: UnpublishedDashletProps) {
                   </Typography>
                 }
               />
+              {over === item.path && <DashletItemOptions path={item.path} />}
```
Comment (rart) [inline ui/app/src/components/UnpublishedDashlet/UnpublishedDashlet.tsx]: These should be the ListItem `secondaryAction`. Is that not available on `ListItemButton`? Also, the show hide might be achievable via css only so we can remove the hovering events.

File: ui/app/src/components/UnpublishedDashlet/UnpublishedDashlet.tsx [hunk 2]
```diff
@@ -89,6 +96,7 @@ export function UnpublishedDashlet(props: UnpublishedDashletProps) {
   const selectedItems = Object.values(itemsById)?.filter((item) => selected[item.id]) ?? [];
   const selectionOptions = useSelectionOptions(selectedItems, formatMessage, selectedCount);
   const isIndeterminate = hasSelected && !isAllSelected;
+  const [over, setOver] = useState(null);
```
Comment (rart) [inline ui/app/src/components/UnpublishedDashlet/UnpublishedDashlet.tsx]: If this is kept, rename appropriately (e.g. `currentHoverPath`)

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/UnpublishedDashlet/UnpublishedDashlet.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/UnpublishedDashlet/UnpublishedDashlet.tsx]: yeah `secondaryAction` is not available in `ListItemButton`

File: ui/app/src/components/ExpiringDashlet/ExpiringDashlet.tsx [hunk 3]
```diff
@@ -58,7 +66,15 @@ const renderExpiredItems = (
...
     return (
-      <ListItem key={index}>
+      <ListItem
+        key={index}
+        sx={{ '&:hover .dashlet-item-options': { display: 'inline-block !important' } }}
+        secondaryAction={
+          <Box className="dashlet-item-options" display="none">
+            <DashletItemOptions path={item.sandboxItem.path} />
+          </Box>
+        }
```
Comment (rart) [inline ui/app/src/components/ExpiringDashlet/ExpiringDashlet.tsx]:
```suggestion
        sx={{ [`.${classes.secondaryAction}`]: { display: 'none' }, [`&:hover ${classes.secondaryAction}`]: { display: 'inline-block' } }}
        secondaryAction={
          <DashletItemOptions path={item.sandboxItem.path} />
        }
```

Comment (rart) [reply ui/app/src/components/ExpiringDashlet/ExpiringDashlet.tsx]: Apply to all

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]

**Pull request merged**